gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskTest.groovy

```diff
@@ -182,5 +272,15 @@ class CreateManifestTaskTest extends GradleTestSpec {
                 CreateManifestTaskTest.class.getResourceAsStream("/b-1.0.jar"),
                 new File(mavenRepo, "b/b/1.0/b-1.0.jar").toPath(),
                 StandardCopyOption.REPLACE_EXISTING)
+        // Make d.jar a duplicate of b.jar, including the exact same recommendation
+        Files.copy(
+                CreateManifestTaskTest.class.getResourceAsStream("/b-1.0.jar"),
+                new File(mavenRepo, "d/d/1.0/d-1.0.jar").toPath(),
+                StandardCopyOption.REPLACE_EXISTING)
+        // Make e.jar a duplicate of b.jar, but with a different recommendation
```

Comment:
if this grows over time, may be worth calling these more sane names, like `dupeOfB` or something